pyHS100/pyHS100.py

```diff
@@ -316,12 +492,20 @@ def query(host, request, port=9999):
 
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         sock.connect((host, port))
+        _LOGGER.debug("> (%i) %s", len(request), request)
         sock.send(TPLinkSmartHomeProtocol.encrypt(request))
-        buffer = sock.recv(4096)[4:]
+        buffer = bytes()
+        while True:
+            chunk = sock.recv(4096)
+            buffer += chunk
+            #_LOGGER.debug("Got chunk %s".format(len(chunk)))
```

Comment:
Really? Can fix though, I don't see it helping much in case of such debug messages, which may sometimes be useful to have. Better strip it off completely?